python/mxnet/gluon/estimator/estimator.py

```diff
@@ -242,7 +242,9 @@ def fit(self, train_data,
                     self.train_stats['batch_' + loss_metric.name] = loss_metric.get()[1]
 
                 try:
-                    self.train_stats['step'] = "{}/{}".format(batch_size * (i + 1), len(train_data._dataset))
+                    completed_samples = len(train_data._dataset) if i == len(train_data._dataset) - 1 \
```

Comment:
a comment for readability?

Reply:
Done. 568d5dfaa260a8e3c07bc36c53ae334fc7c64743